### DISCUSSION TITLE
请问数据表格未获取到数据时怎么把错误码code一起显示出来？

### DISCUSSION THREAD

**Discussion by SouSuoKe** [layui/layui#2113] [问答] [answered]
后端定义了很多错误码，但是对应了很多相同的错误信息给前台显示，测试中发现无法获取数据时不能同时显示错误码，只显示了错误信息，请问如何操作能把错误码一起显示出来？
错误示例如图，好多个地方都写了请重新登录，不想把具体报错信息显示在前台，如果把错误码也显示出来才能更方便的定位问题：
![Snipaste_2024-07-18_16-16-09-889](https://github.com/user-attachments/assets/3bda592a-a2db-4592-8a74-c1672fea08ef)

**Comment by sunxiaobin89** [comment, marked as answer]
目前没有可以自定义 format msg 的途径，建议通过 parseData 进行数据处理，异常的时候将想要显示的效果返回即可，比如：
```
parseData: function (res) {
  // res 即为原始返回的数据
  if (res.code !== 0) {
    res.msg = res.code + ':' + res.msg
  }
  return res;
},
```

**Reply by SouSuoKe** [reply]
好用~~~
